### ISSUE TITLE
Look into MathML integration with Wikidata

### ISSUE THREAD

**Issue by Daniel-Mietchen** [WDscholia/scholia#444]
From the conclusion section of [Representing Mathematical Formulae in Content MathML using Wikidata](https://www.wikidata.org/wiki/Q55979253):

> The  Wikidata  macros  in  LaTeX  are
easy to write and read and thus adequate for editors who are not familiar with
the syntax and structure of MathML. It will be an important mission to con-
vince researchers in the mathematical sciences of the benefits of making their
published content machine-readable. In their documents, they should mark as
many entities as possible:
> - named entities, e.g.
\w{Q210546}{Equivalence principle}
> - whole formulae, e.g.
\w{Q210546}{$E=mc^2$}
> - parts of formulae, e.g.
$\w{Q11379}{E} = \w{Q11423}{m} \w{Q2111}{c}^2$
>
> We are planing to develop a GUI to facilitate the annotation process. This will
produce labeled data to use for the future discovery and recognition of formula
concepts, our long-term research goal.

Perhaps we should coordinate with them about these GUI aspects.

**Comment by physikerwelt** [comment]
@Daniel-Mietchen I looked at the file. Is the intention that one can just use \cite{Qxxx} within LaTeX and the rest is taken care of by the by the tex.py script? That's something I always wanted to have.

**Comment by fnielsen** [comment]
Yes. You can see an example here: https://github.com/fnielsen/scholia/wiki

See also section 4 in https://link.springer.com/content/pdf/10.1007%2F978-3-319-70407-4_36.pdf

It is not completely implemented, e.g., wrt. UTF-8 encoding and books, chapters, etc.
